### PR TITLE
Register achintya.is-a.dev

### PR DESCRIPTION
**Pull request by Moon4546X**
Register achintya.is-a.dev with TXT record pointing to moon061005.

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]
maskduck no like this PR

File: domains/achintya.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Moon4546X",
+           "email": "",
+           "discord": "713352608172998707"
+        },
+    
+        "record": {
+            "TXT": "918ba5486b5908dce20187856dbefd"
```
Comment (DEV-DIBSTER) [inline domains/achintya.json:9]: TXT file alone doesn't do anything.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/achintya.json (hunk 1 above)
Comment (Moon4546X) [reply domains/achintya.json:9]: this is my first time doing this, I am having trouble what should I add next?

Comment (CuteDog5695) [reply domains/achintya.json:9]: Do you have a website at least?

Comment (Moon4546X) [reply domains/achintya.json:9]: yes I do

Comment (CuteDog5695) [reply domains/achintya.json:9]: Then you can check https://is-a.dev/docs to see what records we support and what to do.

**Comment by orxngc** [comment]
Closing as stale.

**Comment by orxngc** [comment]
Wait no it's not stale mb

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**